kafka-rest/src/test/java/io/confluent/kafkarest/integration/AbstractConsumerTest.java

```diff
@@ -96,6 +96,10 @@ protected String consumerNameFromInstanceUrl(String url) {
    * @return the new consumer instance's base URI
    */
   protected String startConsumeMessages(String groupName, String topic, EmbeddedFormat format,
+      String expectedMediatype) {
+    return startConsumeMessages(groupName, Collections.singletonList(topic), format, expectedMediatype);
+  }
+  protected String startConsumeMessages(String groupName, List<String> topics, EmbeddedFormat format,
```

Comment:
added overloaded method to allow for a consumer to consume from multiple topics (initially needed for lag-related integration tests, but eventually removed the dependency on AbstractConsumerTest)